adapters/postgres/postgres.go

```diff
@@ -155,9 +155,17 @@ func OrderByRequest(r *http.Request) (string, error) {
 	return values, nil
 }
 
+// CountByRequest returns true if should use COUNT function in query
+func CountByRequest(r *http.Request) bool {
```

Comment:
@felipeweb how do you suggest doing so? i've added a new `statements.SelectCountInTable` constant, which is used if `CountByRequest` returns `true`. do you think that decision should made inside `CountByRequest`?